db/warehouse/migrate/20240118203430_add_scan_card_codes_table.rb

```diff
@@ -0,0 +1,12 @@
+class AddScanCardCodesTable < ActiveRecord::Migration[6.1]
+  def change
+    create_table(:hmis_scan_card_codes) do |t|
+      t.references :client, null: false
+      t.string :code, index: true, null: false # code to embed in scan card
+      t.references :created_by # user that generated code
+      t.references :deleted_by # user that deleted code
```

Comment:
```suggestion
      t.string :code, index: true, null: false, comment: 'code to embed in scan card'
      t.references :created_by, comment: 'user that generated code'
      t.references :deleted_by, comment: 'user that deleted code'
```

Why not document this in the DB as well.